manuscript/ha-docker-swarm/traefik.md

```diff
@@ -30,7 +30,7 @@ To deal with these gaps, we need a front-end load-balancer, and in this design,
 
 While it's possible to configure traefik via docker command arguments, I prefer to create a config file (`traefik.toml`). This allows me to change traefik's behaviour by simply changing the file, and keeps my docker config simple.
 
-Create `/var/data/traefikv1/traefik.toml` as follows:
+Create `/var/data/traefik/traefik.toml` as follows:
```

Comment:
`/var/data/traefikv1/` is actually the correct path in this case, because I'm about to introduce a companion recipe for traefik v2 :)

Reply:
I figured that was the reason - but then shouldn't we be renaming everything `traefikv1` - i.e. under `/data` and `/config` and even the compose files `traefikv1-app.yml`?

Would it make sense keep the directory `/traefik` and just rename the `toml` file based on version?

Reply:
Scratch that, reverted the directory name and fixed all references (and fixed a couple of others)